fdbserver/DataDistribution.actor.cpp

```diff
@@ -2569,6 +2578,186 @@ ACTOR Future<Void> waitUntilHealthy(DDTeamCollection* self, double extraDelay =
 	}
 }
 
+// This function is very expensive to call. Use with caution.
+ACTOR Future<Void> printSnapshotTeamsInfo(Reference<DDTeamCollection> self, std::string tcIdString) {
+	state DatabaseConfiguration configuration;
+	state std::map<UID, Reference<TCServerInfo>> server_info;
+	state ServerStatusMap server_status;
+	state vector<Reference<TCTeamInfo>> teams;
+	state std::map<Standalone<StringRef>, Reference<TCMachineInfo>> machine_info;
+	state std::vector<Reference<TCMachineTeamInfo>> machineTeams;
+	state std::vector<std::string> internedLocalityRecordKeyNameStrings;
+	state int machineLocalityMapEntryArraySize;
+	state std::vector<Reference<LocalityRecord>> machineLocalityMapRecordArray;
+	state int traceEventsPrinted = 0;
+	state std::vector<const UID*> uids;
+	loop {
+		wait(self->printDetailedTeamsInfo.onTrigger() || delay(SERVER_KNOBS->DD_TEAMS_INFO_PRINT_INTERVAL));
+
+		traceEventsPrinted = 0;
+		if (now() - self->lastTeamsInfoSnapshotTime >= SERVER_KNOBS->DD_TEAMS_INFO_SNAPSHOT_REFRESH_INTERVAL) {
+			// Refresh state snapshot
+			self->lastTeamsInfoSnapshotTime = now();
```

Comment:
I prefer  to replace lastTeamsInfoSnapshotTime with "lastTimeTraceTeamCollectionDetails", which IMHO is more descriptive.

Reply:
`lastTeamsInfoSnapshotTime` means exactly what the name suggests: it means when was the last time the local snapshot of all necessary data was copied from the `DDTeamCollection` instance.

`lastTimeTraceTeamCollectionDetails` means something else.

Reply:
Following your description,
 `lastTeamCollectionInfoSnapshotTime `  sounds better.

Reply:
Done.